generic3g/specs/VariableSpec.F90

```diff
@@ -14,14 +14,16 @@ module mapl3g_VariableSpec
    use mapl_ErrorHandling
    use mapl3g_StateRegistry
    use mapl3g_StateItem
+   use mapl3g_EsmfRegridder, only: EsmfRegridderParam
+   use mapl3g_FieldDictionary
    use esmf
    use gFTL2_StringVector
    use nuopc
 
    implicit none
    private
 
-   public :: VariableSpec
+   public :: VariableSpec, get_regrid_method_from_field_dict
```

Comment:
```suggestion
   public :: VariableSpec
   public :: get_regrid_method_from_field_dict
```
This method should ultimately go into a different module that extracts things from the field dict.   The name will improve on that change.

Reply:
Done. Now that FieldSpec is not calling `get_regrid_method_from_field_dict`, we don't need to make it public.